src/Microsoft.Data.SqlClient/src/Microsoft/Data/Common/DbConnectionStringCommon.cs

```diff
@@ -919,18 +734,18 @@ internal static SqlConnectionColumnEncryptionSetting ConvertToColumnEncryptionSe
         /// <returns></returns>
         internal static bool TryConvertToAttestationProtocol(string value, out SqlConnectionAttestationProtocol result)
         {
-            if (StringComparer.InvariantCultureIgnoreCase.Equals(value, AttestationProtocolHGS))
+            if (StringComparer.InvariantCultureIgnoreCase.Equals(value, nameof(SqlConnectionAttestationProtocol.HGS)))
```

Comment:
I like the `nameof(thing)` rather than having extra constants 👍

Reply:
I like to gain benefit of strong types. 😉